bin/ChakraCore/CMakeLists.txt

```diff
@@ -30,6 +30,7 @@ if(CMAKE_SYSTEM_NAME STREQUAL Linux)
   set(LINKER_END_GROUP
     -Wl,--no-whole-archive
     -Wl,--end-group
+    -static-libstdc++
```

Comment:
Does this line means "libChakraCore.so" always statically linked to stdc++? Did you observe significant size increase?

When you say "However, embedders of ChakraCore are free to decide whether they want to link the dependencies statically or not as before", is that option only for static libs, since no customization exists for "libChakraCore.so"?